css/ampshared.css

```diff
@@ -449,8 +449,13 @@ amp-accordion > section {
 }
 
 /* Collapse content by default. */
-amp-accordion > section > :last-child {
-  display: none !important;
+amp-accordion > section:not([expanded]) > :last-child {
```

Comment:
Ideally we'd not change the `ampshared.css`, but would leave all changes to the actual extension. E.g. the extension would override the initial CSS. This would make sense for several reasons:
 1. We want to keep shared css as small as possible - it's only there to keep things sensible until we upgrade the elements from the actual extension.
 2. We will be running an experiment for this feature for some time, so it'd be better to keep it maximally configurable from the extension itself.
 3. There's not a lot of value in setting up display locking here before we actually have a chance to set up the "beforeactivate" listener. Which means waiting for the extension to load.
 4. The final solution will involve a bit more than just `position:absolute`. We rely on `display:none` for default resource management behavior. In cases when we can't use `display:none`, this management has to be done manually. Which again means waiting for the extension to load.